layers/+lang/lean/packages.el

```diff
@@ -0,0 +1,42 @@
+;;; packages.el --- Lean Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2014 Sylvain Benner
+;; Copyright (c) 2014-2015 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+(setq lean-packages '(mmm-mode 
+                      f 
+                      dash-functional
```

Comment:
@person808 are you sure about `f`. I also tried making my layer not own it, but then its not installed, so Spacemacs probably doesn't install it.